js/pageController.js

```diff
@@ -0,0 +1,29 @@
+var pageViewController = {};
+
+pageViewController.toggleMoblieMenu = function() {
+  $('#pageHeader i').on('click', function(){
+
+    if ($('#pageHeader nav').css('display') === 'none') {
```

Comment:
There might be an easier way to write this with less code you might be able to just check if the  property exists.